x-pack/plugins/cloud_security_posture/public/common/api/use_cis_kubernetes_integration.tsx

```diff
@@ -21,6 +22,8 @@ export const useCisKubernetesIntegration = () => {
   const { http } = useKibana().services;
 
   return useQuery<GetInfoResponse, DefaultPackagesInstallationError>(['integrations'], () =>
-    http.get<GetInfoResponse>(epmRouteService.getInfoPath(CLOUD_SECURITY_POSTURE_PACKAGE_NAME))
+    http.get<GetInfoResponse>(epmRouteService.getInfoPath(CLOUD_SECURITY_POSTURE_PACKAGE_NAME), {
+      version: API_VERSIONS.public.v1,
```

Comment:
what I don't understand why the e2e didn't fail when this was introduced.

Reply:
[it looks like](https://elastic.slack.com/archives/C04HT4P1YS3/p1694710887394809) the check only enabled in local dev, therefore tests don't catch it

Reply:
More detailed explanation in [this discussion](https://elastic.slack.com/archives/C04HT4P1YS3/p1695124175172019?thread_ts=1694710887.394809&cid=C04HT4P1YS3)